dev-python/networkx/networkx-1.1.recipe

```diff
@@ -0,0 +1,78 @@
+SUMMARY="Simple DNS resolver for asyncio"
+DESCRIPTION="aiodns provides a simple way for doing asynchronous DNS \
+resolutions with a synchronous looking interface by using pycares"
+HOMEPAGE="https://pypi.python.org/pypi/aiodns"
+COPYRIGHT="2016 Saúl Ibarra Corretgé"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="040052202a9776fc9979e414a6d5244718a5ad57"
+SOURCE_URI="https://github.com/networkx/networkx/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="ee179fcf853731c4fb399639ab6e989f8e09659454b9343594fa699d5835d154"
+SOURCE_DIR="networkx-$srcGitRev"
+
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	$portName = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+
+PYTHON_PACKAGES=(python3)
+PYTHON_VERSIONS=(3.6)
+for i in "${!PYTHON_PACKAGES[@]}"; do
+	pythonPackage=${PYTHON_PACKAGES[i]}
+	pythonVersion=${PYTHON_VERSIONS[$i]}
+
+	eval "PROVIDES_$pythonPackage=\"
+		${portName}_$pythonPackage = $portVersion
+		\""
+	eval "REQUIRES_$pythonPackage=\"
+		haiku
+		cmd:python$pythonVersion
+		\""
+	BUILD_REQUIRES="$BUILD_REQUIRES
+		setuptools_$pythonPackage"
+	BUILD_PREREQUIRES="$BUILD_PREREQUIRES
+		cmd:python$pythonVersion"
```

Comment:
I'd move these 4 lines:
```
	BUILD_REQUIRES="$BUILD_REQUIRES
		setuptools_$pythonPackage"
	BUILD_PREREQUIRES="$BUILD_PREREQUIRES
		cmd:python$pythonVersion"
```
outside of the `for i` loop. (It is slightly better IMHO.)
And I'd also add support for python 2.7. See:
https://github.com/haikuports/haikuports/blob/1a84a3ba03ef70f8b5b8d8d1ce81a095bb9528bb/dev-python/tzlocal/tzlocal-1.5.1.recipe#L22